lib/coinbase/validator.rb

```diff
@@ -0,0 +1,82 @@
+# frozen_string_literal: true
+
+module Coinbase
+  # A representation of a staking validator.
+  class Validator
+    # Returns a new Validator object.
+    # @param model [Coinbase::Client::Validator] The underlying Validator object
+    def initialize(model)
+      @model = model
+    end
+
+    # Returns a list of Validators for the provided network and asset.
+    # @param network_id [Symbol] The network ID
+    # @param asset_id [Symbol] The asset ID
+    # @param status [Symbol] The status of the validator. Defaults to nil.
```

Comment:
noticde that some validators in your example have `unknown` status.

Reply:
Yes - currently that's what the backend returns. The beacon chain can take hours to process the broadcasted deposit tx on execution layer only after which the state can proceed ahead. We need to add more intelligence in our staking api backend to detect the deposit on the execution chain and add an intermediary state like `depositing`